protected/models/Patient.php

```diff
@@ -311,6 +311,30 @@ public function getPatientSource()
         return 'None';
     }
 
+    /**
+     * @param $status string Patient's status in a trial.
+     * @return string Number of trials the patient is in where they have the given status within a trial. This is returned as a string to preserve max. precision.
+     */
+    public function getTrialCount($status)
```

Comment:
Try to move this into the trial submodule where applicable. We want to try and keep the core untainted.